test/Combinatorics/EnumerativeCombinatorics/multipartitions.jl

```diff
@@ -0,0 +1,41 @@
+@testset "Multipartitions" begin
```

Comment:
Ideally the tests would be run twice, once for `Int` and once for `ZZRingElem`. For starters:
```suggestion
@testset "Multipartitions" for T in [Int, ZZRingElem]
```
and then adjust tests to use `T`, e.g. change `[]` or `[3,2]` etc. to `T[]`, `T[3,2]` etc.